rls/src/main/java/io/grpc/rls/AdaptiveThrottler.java

```diff
@@ -206,7 +206,7 @@ void increment() {
     }
 
     // Represents a slot which is not initialized and is unusable.
-    private static final Slot NULL_SLOT = new Slot(-1);
+    private static final Slot NULL_SLOT = new Slot(Long.MIN_VALUE);
```

Comment:
Since Platform.systemNanoTime() can return negative values per the spec, the logic for identifying relevant slots that is assuming that the NULL_SLOT.endNanos is less than "now" is broken when -1 is used.  It isn't possible for a value to be less than Long.MIN_VALUE, so all values of "now" will work as expected if that is used.

Reply:
Okay, so that demonstrates this PR is insufficient. `nanoTime()` is permitted to overflow, so the only valid way to compare `nanoTime()` values is using subtraction.

Reply:
Changed to use subtraction.